mz_crypt_openssl.c

```diff
@@ -494,7 +494,7 @@ typedef struct mz_crypt_hmac_s {
     EVP_MAC_CTX *ctx;
 #endif
     int32_t     initialized;
-    int32_t     error;
+    unsigned long     error;
```

Comment:
Can you fix the white space alignment of the other variables in the struct to match up?